rasa/core/test.py

```diff
@@ -799,6 +791,16 @@ async def test(
                 targets, predictions, output_dict=True
             )
 
+            # Add conversation level accuracy to story report.
+            num_failed = len(story_evaluation.failed_stories)
+            num_correct = len(story_evaluation.successful_stories)
+            num_convs = num_failed + num_correct
+            conv_acc = num_correct / num_correct if num_correct else 0.0
```

Comment:
This seems like a typo? num_correct/num_correct == 1

Reply:
Also GH won't let me comment on the particular line because it's too far out of scope, but while you're at it would you remove the unused import on line 670? (`from rasa.test import get_evaluation_metrics`)

Reply:
Oof that's embarrassing! Thank you for catching! Will fix!